Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/InteractionHint.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    public class InteractionHint : MonoBehaviour
+    {
+        private GameObject VisualsRoot;
+
+        [Tooltip("React to hand tracking state to hide visuals when hands are being tracked.  If false, only the customShouldHideHands function will be evaluated.")]
```

Comment:
The class is named InteractionHint but the field descriptions are all specific to hand interactions. Should we rename this to HandInteractionHint?